tests/providers/docker/decorators/test_docker.py

```diff
@@ -100,3 +104,29 @@ def do_run():
 
         assert len(dag.task_ids) == 21
         assert dag.task_ids[-1] == "do_run__20"
+
+    @pytest.mark.parametrize(
+        "extra_kwargs, actual_exit_code, expected_state",
+        [
+            (None, 99, TaskInstanceState.FAILED),
+            ({"skip_exit_code": 100}, 100, TaskInstanceState.SKIPPED),
+            ({"skip_exit_code": 100}, 101, TaskInstanceState.FAILED),
+            ({"skip_exit_code": None}, 0, TaskInstanceState.SUCCESS),
+        ],
+    )
+    def test_skip_docker_operator(self, extra_kwargs, actual_exit_code, expected_state, dag_maker):
+        @task.docker(image="python:3.9-slim", **(extra_kwargs if extra_kwargs else {}))
+        def f(exit_code):
+            exit(exit_code)
```

Comment:
```suggestion
            raise SystemExit(exit_code)
```
Better use [sys.exit()](https://docs.python.org/3/library/sys.html#sys.exit) or raise SystemExit(), if you don't wont to import `sys`, rather then [exit()](https://docs.python.org/3/library/constants.html#constants-added-by-the-site-module)